2_phase_minting_schedule/main.tex

```diff
@@ -4,61 +4,62 @@
-\\\\
-\textit{Dangers to operational feasibility, incentives and pricing}
+Hence this model attempts to mitigate the risk of \textbf{fledgling network demand coinciding with a disincentivized, dwindling or transient supply}. Note that these arguments do not involve strong claims about when (or if) network adoption will occur. Instead, it is the acknowledgement of uncertainty with respect to an adoption timeline that forms the core of the problem, and underpins the utility and advantage of a stable issuance phase as part of a two-phase minting schedule. 
+
+\subsection{Impeding centralization}
+
+A stable issuance in the first phase brings other benefits, including the partial mitigation of a centralizing dynamic suffered by many staking networks, in which larger node operators utilize subsidies to increase/consolidate control of the token supply, thanks to the allocation of tokens in proportion to stake size. Moreover, larger operators can re-stake a greater percentage of subsidies (not needing to convert as much to fiat to cover operational costs), precipitating a `rich-get-richer' dynamic. If token issuance is highest in the earliest days of the network, larger operators are granted an even greater head-start over new node operators who start staking in a later era, \textbf{the presence of whom provides the network with an important, long-term counter-balance against centralization}. Relatedly, an important rationale for the \textit{work token} model is the supposition that new node operators purchase the network's native token in order to stake and earn revenue. With a token issuance that decays from genesis, new operators are doubly disadvantaged – they must pay the market price to acquire tokens, plus they receive a diminished subsidy. 
 \\\\
-As mentioned, the two-phase model implies a lower genesis inflation rate than the previous draft model. This itself confers further advantages to network health:  
+Hence, a two-phase minting schedule assists all would-be node operators who don't happen to stake right at network launch, but decide to do so at a later date. By addressing the temporal inequity of the previous draft, we impede problematic centralization trends.
+
+\subsection{Risks to operational feasibility, incentives and pricing}
+
+As mentioned, the two-phase model implies, relative to other subsidy models, a lower number of tokens minted on day one. This in itself confers further advantages to network health:  
```

Comment:
This is not really clear since even with two-phase inflation, the amount of tokens minted on day one is the highest it will ever be.

Reply:
```suggestion
As Figure 1 shows, a `generic' subsidy model involves high issuance at genesis. In the two phase model, the number of tokens minted per day isn't only stable in phase one, it also commences and remains a lower figure for the first few years. This difference confers further benefits to network health:  
```
Agree, the use of 'day one' is unnecessary.